### DISCUSSION TITLE
Table of Contents for Vercel Rest API Endpoints is Broken

### DISCUSSION THREAD

**Discussion by versecafe** [vercel/community#7034] [Bug Report]
### Page to Investigate

https://vercel.com/docs/rest-api/endpoints

### Steps to Reproduce

<img width="1394" alt="image" src="https://github.com/vercel/community/assets/147033096/30dd9e55-e7fb-483f-b009-f5c43d283c6d">

**Comment by amyegan** [comment]
Thanks for reporting this! The engineering team is investigating
